helper-install-puppeteer-deps

```diff
@@ -10,7 +10,8 @@ set -eu -o pipefail
 
 # HELPER COMMANDS
 
-sudo apt-get update && \
+sudo apt-get update
```

Comment:
Does `&& \` need to be removed here? Wouldn't we still want to fail if updating fails?

Reply:
> Does && \ need to be removed here?

Not really besides legibility.

> Wouldn't we still want to fail if updating fails?

It would! We only needed the `&&` as for some reason we were updating and installing in one call.

That's not really necessary in a script, where I'd aim to avoid unless it's necessary.

See https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/, we're setting `set -eu -o pipefail`.

Reply:
Ah fair enough, as long as it's intentional.

Good read 👍